static/js/controllers/contacts.js

```diff
@@ -219,6 +224,63 @@ var _ = require('underscore'),
         $scope.search();
       };
 
+      $scope.simprintsEnabled = !!(window.medicmobile_android &&
+          window.medicmobile_android.simprints_available &&
+          window.medicmobile_android.simprints_available());
+      $scope.triggerSimprintsIdent = function() {
```

Comment:
There are many ways you can "launch the fingerprint scanner".  However, we're specifically launching it for _identification_.  (as opposed: registration, update, verification, adding metadata). This is the same language used in the simprints API, and I think being consistent with this language even in the webapp is probably helpful.  Perhaps some developer one day won't know what `simprints` refers to, but to me it's clearer than using `fingerprint-scanner` everywhere.  We don't necessarily refer to other libs or services by generic names, e.g. pouchdb is not "client-side-storage" throughout our app.